source/getting-started/tutorials/sensors/index.md

```diff
@@ -191,15 +190,15 @@ Run Mean:	10.25 usec
 Run Var:	1.40 usec
 ```
 
-The loop time has returned to 100.00 usec. The timing manager is not slowing down the rate of the control task anymore.
+The Loop Mean has returned to 100.00 usec. The Timing Manager is not slowing down the rate of the control task anymore.
 
-But the Run time has increased significantly. Why is that?
+But the Run-Time has increased significantly. Why is that?
 
 The answer is that I have no idea. Ask Patrick
```

Comment:
Before merging, we should remove this (although it is funny). It would be more professional to say something like:

"However, the task's Run-Time has increased significantly. This is a bug under review that may appear from sub-optimal timing configuration."